src/core/namespace.c

```diff
@@ -538,6 +628,21 @@ static unsigned int namespace_calculate_mounts(
                 ProtectHome protect_home,
                 ProtectSystem protect_system) {
         unsigned n;
+        unsigned protect_system_cnt = 0;
```

Comment:
Maybe it'd be nicer to do
```
    protect_system_count =
                protect_system == PROTECT_SYSTEM_COUNT ? ELEMENTSOF(protect_system_strict_table) :
                protect_system == PROTECT_SYSTEM_YES ? ELEMENTSOF(protect_system_yes_table) :
                protect_system == PROTECT_SYSTEM_FULL ? ELEMENTSOF(protect_system_yes_table) : 0;
```
I know it repeats `protect_system` a bit, but swith is annoyingly verbose.